### ISSUE TITLE
Update Progressbar During Validation

### ISSUE THREAD

**Issue by dgrahn** [keras-team/keras#13527]
**System information**  
N/A

**Describe the current behavior**  
If the validation stage for each epoch is non-trivial and takes more than a few seconds, the progress bar appears to be stuck with an ETA of 0s.

**Describe the expected behavior**  
Instead, the progress bar should change to progressing through validation data batches and provide an estimate of how long the validation should take.

**Code to reproduce the issue**  
N/A

**Other info / logs**  
N/A

**Comment by jamesbraza** [comment]
Sorry for grave digging this, but can you point to docs in Keras where one can get a progress bar when evaluating the validation dataset?

I see this issue was closed as completed, but it's not clear what about it was completed.

For what it's worth, I am invoking `Model.fit` with `tensorflow==2.4.4`, but do not have a validation progress bar:

```python
model.fit(
    train_data_generator,
    epochs=MAX_NUM_EPOCHS,
    steps_per_epoch=steps_per_epoch,
    validation_data=val_data_generator,
    validation_steps=VALIDATION_STEPS,
    callbacks=[callbacks]
)
```

**Comment by tgpsantos** [comment]
I'm also searching for something like this because validation data can take up a lot of time in large training datasets, and it'd be nice to know its progress.

I believe atm one can circumvent this by removing validation ds from model.fit and either:
- use on_epoch_end callback
- calling model.fit for every epoch with a model.evaluate

**Comment by aamster** [comment]
@tgpsantos yes, but then we lose the ability to use `ModelCheckpoint` callback when evaluating the validation set?
